packages/circle-demo-webapp/app/routes/wallet.$id/route.tsx

```diff
@@ -31,31 +29,34 @@ export async function loader({ params }: LoaderFunctionArgs) {
     throw new Error('Wallet ID is required');
   }
 
-  const [balancesRes, walletRes, transactionsRes] = await Promise.all([
+  const [balancesRes, walletRes] = await Promise.all([
     sdk.getWalletTokenBalance({
       id,
       includeAll: true,
     }),
     sdk.getWallet({ id }),
-    sdk.listTransactions({
```

Comment:
why you don't like getting initial transactions from the server? loading them on the client side slows down the page load and requires you to handle loading and error states.

Reply:
we need to add Tokens to the transaction. we already have this functionality in another place. we have two choices:
- use what we have (fast way)
- make a separate helper to fill transactions with tokens